tests/io/v3/was/test_attachments_api.py

```diff
@@ -0,0 +1,118 @@
+'''
+Testing the Attachments endpoints
+'''
+import os
+import re
+
+import responses
+
+ATTACHMENTS_BASE_URL = r'https://cloud.tenable.com/api/v3/was/attachments'
+USERS_API_ID = r'([0-9a-fA-F\-]+)'
+
+dir_name = os.path.dirname(os.path.abspath(__file__))
+
+IMG_ATTACHMENT_FILE = 'img_attachment_file.png'
+TXT_ATTACHMENT_FILE = 'text_attachment_file.txt'
+
+IMG_ATTACHMENT_FILE_PATH = os.path.join(dir_name, IMG_ATTACHMENT_FILE)
+TXT_ATTACHMENT_FILE_PATH = os.path.join(dir_name, TXT_ATTACHMENT_FILE)
+
+IMG_ATTACHMENT_ID = 'b1b335cf-938f-462a-b83d-088b72428a2a'
+TXT_ATTACHMENT_ID = '49484c93-baa5-4952-8043-9afad238cb7b'
+
+IMG_CONTENTS = None
+TXT_CONTENTS = None
+
+
+@responses.activate
+def test_download_img_attachment(api):
+
+    global IMG_CONTENTS
+
+    if IMG_CONTENTS is None:
+        with open(IMG_ATTACHMENT_FILE_PATH, 'rb') as img_file:
+            IMG_CONTENTS = img_file.read()
+
```

Comment:
can we try to write this logic in a separate method, as we are using it at multiple place

Reply:
Done.